usaspending_api/etl/transaction_loaders/data_load_helpers.py

```diff
@@ -0,0 +1,110 @@
+import datetime
+import os
+import re
+import boto3
+import csv
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger("console")
+
+
+def capitalize_if_string(val):
+    try:
+        return val.upper()
+    except AttributeError:
+        return val
+
+
+def false_if_null(val):
+    if val is None:
+        return False
+    return val
+
+
+def format_value_for_sql(val, cur):
+    return str(cur.mogrify("%s", (val,)), "utf-8")
+
+
+def format_bulk_insert_list_column_sql(cursor, load_objects, type):
+    """creates formatted sql text to put into a bulk insert statement"""
+    keys = load_objects[0][type].keys()
+
+    columns = ['"{}"'.format(key) for key in load_objects[0][type].keys()]
+    values = [[format_value_for_sql(load_object[type][key], cursor) for key in keys] for load_object in load_objects]
+
+    col_string = "({})".format(",".join(columns))
+    val_string = ",".join(["({})".format(",".join(map(str, value))) for value in values])
+
+    return col_string, val_string
+
+
+def format_insert_or_update_column_sql(cursor, load_object, type):
+    """creates formatted sql text to put into a single row insert or update statement"""
+    columns = []
+    values = []
+    update_pairs = []
+    for key in load_object[type].keys():
+        columns.append('"{}"'.format(key))
+        val = format_value_for_sql(load_object[type][key], cursor)
+        values.append(val)
+        if key not in ["create_date", "created_at"]:
+            update_pairs.append(" {}={}".format(key, val))
+
+    col_string = "({})".format(",".join(map(str, columns)))
+    val_string = "({})".format(",".join(map(str, values)))
+    pairs_string = ",".join(update_pairs)
+
+    return col_string, val_string, pairs_string
+
+
+def get_deleted_fpds_data_from_s3(date):
```

Comment:
Definitely future tech debt work to refactor this function to be cleaner and leverage Python features. Out of scope for this PR